tests/CanadaPost/Tests/NCShipmentTest.php

```diff
@@ -0,0 +1,313 @@
+<?php
+
+namespace CanadaPost\Tests;
+
+use CanadaPost\NCShipment;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * NCShipment Class Tests.
+ *
+ * @group NCShipment
+ */
+class NCShipmentTest extends PHPUnit_Framework_TestCase
+{
```

Comment:
General comment, should consider whether using data providers would help organize your test data. Look at the tests written for the order export.